tests/python/lib/file_utils.py

```diff
@@ -29,12 +29,17 @@ def _check_bin(bin_path):
 _ENGINE_INSTALL_PATH = "/usr/share/engine" if os.environ.get("KPHP_TESTS_INTERGRATION_TESTS_ENABLED") else None
 _DEFAULT_KPHP_REPO = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), "../../../"))
 _KPHP_REPO = os.environ.get("KPHP_TESTS_KPHP_REPO", _DEFAULT_KPHP_REPO)
+_K2_INSTALL_PATH = _KPHP_REPO
 
 
 def search_kphp2cpp():
     return _check_file("objs/bin/kphp2cpp", _KPHP_REPO, _check_bin)
 
 
+def search_k2_bin():
+    return _check_file("objs/bin/k2-run-once", _K2_INSTALL_PATH, _check_bin)
```

Comment:
I think we should not hardcode path to k2 platform. Let's make it a parameter for our tests

Reply:
Yeah, I add param `--k2-bin` for it